lib/dry/auto_inject/strategies/constructor.rb

```diff
@@ -23,6 +24,8 @@ def initialize(container, *dependency_names)
 
         # @api private
         def included(klass)
+          fail Error, "cannot inject dependencies into a module" unless klass.respond_to?(:new)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Prefer single-quoted strings when you don't need string interpolation or special symbols.](https://app.codacy.com/gh/dry-rb/dry-auto_inject/pullRequest?prid=4763762)